js/scripts/Game.js

```diff
@@ -0,0 +1,30 @@
+/**  @type {HTMLCanvasElement} */ 
+
+class Game{
+  constructor(ctx, width, height, car){
+      this.ctx = ctx
+      this.width = width
+      this.height = height
+      this.car = car
+      this.isIntervalID = null;
+      this.frames = 0;
```

Comment:
Inside of the Game constructor, you should've created the image.

```
const backgroundImage = new Image();
backgroundImage.src = "/images/road.png"
this.imgRoad = backgroundImage;
```

and then on the update function, you can draw the image!

`this.ctx.drawImage(this.imgRoad, 0, 0, canvas.width, canvas.height)`